docs/components.md

```diff
@@ -36,22 +36,23 @@ support at this time). While in alpha, breaking changes may be introduced in a
+| [filelog](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver/filelogreceiver)                           |            |                                                                                                     |            |
+| [journald](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver/journaldreceiver)                         |            |                                                                                                     |            |
+| [kafka](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver/kafkareceiver)                               |            |                                                                                                     |            |
+| [kafkametrics](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver/kafkametricsreceiver)                 |            |                                                                                                     |            |
+| [k8s_events](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver/k8seventsreceiver)                      |            |                                                                                                     |            |
+| [mongodbatlas](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver/mongodbatlasreceiver)                 |            |                                                                                                     |            |
+| [sqlquery](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver/sqlqueryreceiver)                         |            |                                                                                                     |            |
+| [statsd](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver/statsdreceiver)                             |            |                                                                                                     |            |
+| [syslog](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver/syslogreceiver)                             |            |                                                                                                     |            |
+| [tcplog](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver/tcplogreceiver)                             |            |                                                                                                     |            |
+| [windowsperfcounters](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver/windowsperfcountersreceiver)   |            |                                                                                                     |            |
```

Comment:
nit: maybe the column is unnecessary longer than needed 
```suggestion
| [windowsperfcounters](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver/windowsperfcountersreceiver) |            |                                                                                                     |            |
```

Reply:
I fixed this during the rebase.